### DISCUSSION TITLE
albaretto, a little CLI utility (+ thanks for figuring out the requests/responses!)

### DISCUSSION THREAD

**Discussion by afparsons** [ezzcodeezzlife/dalle2-in-python#17] [Show and tell]
Great work!

I wrote a simple command line utility to download images from a task ID and then either save them to local filesystem or upload them to Google Drive. It's a one-trick pony (i.e. not a versatile library), but solved a use case I had. https://github.com/afparsons/albaretto

While I did not use `dalle2-in-python`, I definitely took a look at how `dalle2-in-python` formatted its requests and responses. Thank you for figuring everything out for me!

I'm posting this to _Show and tell_ as I assume our audiences might overlap.

**Comment by ezzcodeezzlife** [comment]
Cool project! 🔥
